pkg/build/api/types.go

```diff
@@ -86,6 +95,10 @@ type BuildStatus struct {
 	// Cancelled describes if a cancel event was triggered for the build.
 	Cancelled bool
 
+	// Skipped indicates that this build was marked as skipped. The build will be
```

Comment:
@bparees hmm that makes sense :-) i can use that label instead of start timestamp.